spec/factories.rb

```diff
@@ -0,0 +1,21 @@
+FactoryBot.define do
+
+  factory :developer, class: Developer do
+    email 'mrabc@gmail.com'
```

Comment:
@truongle7793 
we can use `email { Faker::Internet.email }`

Ref:
https://github.com/stympy/faker#contents

Reply:
Thanks
I have made change it to use Faker.